cli/src/version.js

```diff
@@ -8,13 +8,7 @@ const runCommand = () => {
   console.info(package_json.version);
```

Comment:
if `create-cert` enforces that there are no extra arguments, shouldn't `version` as well?